django_x509/admin.py

```diff
@@ -1,7 +1,17 @@
 from django.contrib import admin
 
-from .base.admin import CaAdmin, CertAdmin
+from .base.admin import AbstractCertAdmin, AbstractCertForm, CaAdmin
 from .models import Ca, Cert
 
+
+class CertForm(AbstractCertForm):
+    class Meta(AbstractCertForm.Meta):
+        model = Cert
+
+
+class CertAdmin(AbstractCertAdmin):
+    form = CertForm
+
+
```

Comment:
You have to do the same for `Ca` because also certification authorities can be imported.

In the truth, if you want to import some certificates you will have to import the certification authority first otherwise as per current implementation it won't work.